dbt/adapters/spark/impl.py

```diff
@@ -216,60 +271,72 @@ def get_columns_in_relation(self, relation: Relation) -> List[SparkColumn]:
             ),
             None,
         )
+
```

Comment:
Given the issues we've seen around column caching (https://github.com/dbt-labs/dbt-spark/issues/431), I think we need to fully disable pulling from the cache in `get_columns_in_relation`. That is, until we're ready to roll out column-level caching as a real feature. That would require us to add cache invalidation logic to macros such as `alter_column_type` and `alter_relation_add_remove_columns`.